### PR TITLE
Add payment examples to the RN app

### PR DESCRIPTION
**Pull request by dangeross**
This PR adds example code on how to (prepare) receive and send lightning payments

### CODE REVIEW

**Review by roeierez — APPROVED** [review]
LGTM

File: packages/react-native/example/App.js [hunk 1]
```diff
@@ -39,27 +51,62 @@ const App = () => {
...
+                // Wait for payer to pay.... once successfully paid an event of `paymentSucceeded` will be emitted.
+                addLine("Bolt11 invoice", `${receivePaymentRes.invoice}`)
+
+                /* Send lightning payment */
+
+                // Set the `bolt11Invoice` to enable sending in the example app
+                if (bolt11Invoice) {
+                    let prepareSendRes = await prepareSendPayment({ invoice: bolt11Invoice })
+                    addLine("prepareSendPayment", JSON.stringify(prepareSendRes))
+                    // Get the fees requred for this payment
```
Comment (erdemyerebasmaz) [inline packages/react-native/example/App.js:102]:
```suggestion
                    // Get the fees required for this payment
```

File: packages/react-native/example/App.js [hunk 2]
```diff
@@ -39,27 +51,62 @@ const App = () => {
...
                 listenerId = await addEventListener(eventHandler)
                 addLine("addEventListener", listenerId)
 
-                let walletInfo = await getInfo()
-                addLine("getInfo", JSON.stringify(walletInfo))
+                /* Receive lightning payment */
+
+                let prepareReceiveRes = await prepareReceivePayment({ payerAmountSat: 1000 })
+                addLine("prepareReceivePayment", JSON.stringify(prepareReceiveRes))
+                // Get the fees requred for this payment
```
Comment (erdemyerebasmaz) [inline packages/react-native/example/App.js:88]:
```suggestion
                // Get the fees required for this payment
```

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

Found a typo

**Pull request merged**